lightning/src/onion_message/messenger.rs

```diff
@@ -563,6 +563,9 @@ where
 		// recipient's node_id.
 		const MIN_PEER_CHANNELS: usize = 3;
 
+		// Add dummy hops to each blinded path to make it more difficult to infer the recipient's position.
+		const DUMMY_HOPS_COUNT: u8 = 3;
```

Comment:
If we always add 3, it doesn't really make anything more difficult? Especially if all impls add three.